pom.xml

```diff
@@ -237,6 +233,14 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.22.2</version>
+                <configuration>
+                    <properties>
+                        <property>
+                            <name>dataproviderthreadcount</name>
+                            <value>1</value>
```

Comment:
Let's try increasing this and run on both iOS and Android

Reply:
@SrinivasanTarget - I tested this with 2 android devices. Do not have an iOS device to try that.

Reply:
I will try this in iOS Simulators and keep you posted.